scripts/Readme.md

```diff
@@ -0,0 +1,30 @@
+# Gen Scripts
+
+This folder contains scripts for setting up Gen and its dependencies. They are tested on Ubuntu 18.04.4 LTS.
+
+Gen can be installed in two ways: Natively on Ubuntu or within Docker. We will describe both ways in dedicated sections.
+
+## Docker
+
+In order to install Docker, please use the script `install_docker.sh` and reboot to add your user to the docker group so you can run Docker without root privileges.
```

Comment:
@marcoct, any thoughts on whether we should assume a Linux machine?  I would guess that many users of this repo will be on Mac or Windows.  Due to the complexity and platform-dependence of the process, it might make sense to point to the official Docker documentation, rather than duplicate here.

Reply:
My take is this: I would never assume or impose Linux, but I like freely sharing among the Linux community as an optional offer. Providing this as an extra seems desirable and easy to maintain - Linux devs usually file PRs if something can be improved or updated. Maybe we should put it into a subfolder `scripts/ubuntu1804`? This way, other communities can easily add their own scripts if desired. Finally, I can essentially guarantee I will write 20.04 LTS scripts as well, so this gets us Ubuntu support until 2025. Alternatively, I can create a dedicated systems repository and push these scripts there.

Reply:
I assume this is resolved. Otherwise, it probably makes sense to just close this PR which is also fine by me - in that case please consider using https://github.com/fplk/mit-ubuntu-scripts/ directly (if you have access).